src/keys/utils.ts

```diff
@@ -538,6 +591,84 @@ async function decryptWithKey(
   return data.buffer.slice(data.byteOffset, data.byteOffset + data.byteLength);
 }
 
+async function writeRootKeyPair(
```

Comment:
Why has this been taken out of `KeyManager`?

Reply:
This should be moved back into `KeyManager`, side-effectful operations should be in the classes, as it relies on the context of the `KeyManager`.

Reply:
Fixed